operator/src/main/resources/scripts/introspectDomain.sh

```diff
@@ -74,6 +75,17 @@ for dir_var in DOMAIN_HOME JAVA_HOME WL_HOME MW_HOME ORACLE_HOME; do
   [ ! -d "${!dir_var}" ] && trace SEVERE "Missing ${dir_var} directory '${!dir_var}'." && exit 1
 done
 
+trace "DATA_HOME=${DATA_HOME}"
+
+#
+# DATA_HOME env variable exists implies override directory specified.  Attempt to create directory
+#
+if [ ! -z "${DATA_HOME}" ] && [ ! -d "${DATA_HOME}" ]; then
+  trace "Creating data home directory: '${DATA_HOME}'"
+  createFolder ${DATA_HOME}
```

Comment:
I wonder if createFolder should be locally defined.   This depends on utils.sh createFolder, which was moved out of startServer.sh to utils.sh and is specific to startServer.sh. It's specific to startServer.sh as it calls 'exitOrLoop' instead of 'exit', plus if it does 'loop' it ultimately exits 0 instead of 1.

Reply:
added back createFolder to be locally defined in introspectDomain.sh with 'exit 1'.